bower.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "polymerfire",
-  "version": "0.9.4",
+  "name": "polymerfire-extended",
+  "version": "0.9.5",
   "authors": [
```

Comment:
This changes are not part of my pull request. I need this changes to start using this component in my app. If i dont change this in the bower file i get the original component without my changes.